src/main/java/bio/terra/service/snapshotbuilder/SnapshotRequestDao.java

```diff
@@ -90,84 +92,34 @@ public SnapshotAccessRequestResponse getById(UUID requestId) {
   /**
    * Return the list of Snapshot Requests associated with the given dataset id.
    *
-   * @param datasetId associated with any number of snapshot requests.
+   * @param authorizedResources snapshot requests that the user has permission to see.
    * @return the list of snapshot requests, empty if none, or an exception if the dataset does not
    *     exist.
    */
   @Transactional(propagation = Propagation.REQUIRED, readOnly = true)
-  public List<SnapshotAccessRequestResponse> enumerateByDatasetId_old(UUID datasetId) {
-    String sql = "SELECT * FROM snapshot_request WHERE dataset_id = :dataset_id";
```

Comment:
Did we decide to push the database changes (to drop `dataset_id` and make `source_snapshot_id ` non-nullable) to a future PR?

Reply:
Yes, pushing that to a future PR for scope reasons